subsys/bluetooth/controller/ll_sw/ull_adv_internal.h

```diff
@@ -50,13 +50,14 @@ const uint8_t *ull_adv_pdu_update_addrs(struct ll_adv_set *adv,
 #define ULL_ADV_PDU_HDR_FIELD_CTE_INFO  BIT(2)
 #define ULL_ADV_PDU_HDR_FIELD_AUX_PTR   BIT(4)
 #define ULL_ADV_PDU_HDR_FIELD_SYNC_INFO BIT(5)
+#define ULL_ADV_PDU_HDR_FIELD_ACAD      BIT(7)
 #define ULL_ADV_PDU_HDR_FIELD_AD_DATA   BIT(8)
 
 /* Helper type to store data for extended advertising
  * header fields and extra data.
  */
-struct adv_pdu_field_data {
-	uint8_t *field_data;
+struct ull_adv_ext_hdr_data {
+	void *field_data;
```

Comment:
Can you elaborate on why this type was changed?

Reply:
to be consistent with `extra_data` below.